pandas/tests/extension/base/ops.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import numpy as np
+import pandas as pd
+from .base import BaseExtensionTests
+
+
+class BaseOpsTests(BaseExtensionTests):
```

Comment:
maybe we can split this in arithmetic and compare ops? So it is easier to fully skip one of the two, depending on what your class implements.

Reply:
I will do that.